src/main/java/org/junitpioneer/jupiter/CartesianEnumSource.java

```diff
@@ -50,17 +49,21 @@
 @Target({ ElementType.ANNOTATION_TYPE, ElementType.METHOD })
 @Retention(RetentionPolicy.RUNTIME)
 @Documented
-@Repeatable(CartesianEnumSource.CartesianEnumSources.class)
+@Repeatable(CartesianEnumSources.class)
 @ArgumentsSource(CartesianEnumArgumentsProvider.class)
 public @interface CartesianEnumSource {
 
 	/**
 	 * The enum type that serves as the source of the enum constants.
 	 *
+	 * <p>If this attribute is not set explicitly, the declared type of the
+	 * parameter of the {@code @CartesianProductTest} method, which has the
+	 * same relative index of the annotation, is used.
```

Comment:
I'm not sure if this is the best way to phrase it. Any suggestions?

Reply:
I think this is OK - if this were a guide, I'd add examples to ease understanding but for Javadoc this is enough. You *could* expand it with an example - if you want to.